From d28c8a472c026e470ddfb223f55be048f654d81a Mon Sep 17 00:00:00 2001
From: Brandon Brnich <b-brnich@ti.com>
Date: Sun, 16 Jun 2024 21:45:44 -0500
Subject: [PATCH 199/222] media: img: vxe-vxd: encoder: Fix Compile Warning

Implicit conversion being done triggering compiler warnings. Add correct
converstion.

Signed-off-by: Brandon Brnich <b-brnich@ti.com>
---
 drivers/media/platform/imagination/vxe-vxd/encoder/vxe_v4l2.c | 7 +++----
 1 file changed, 3 insertions(+), 4 deletions(-)

diff --git a/drivers/media/platform/imagination/vxe-vxd/encoder/vxe_v4l2.c b/drivers/media/platform/imagination/vxe-vxd/encoder/vxe_v4l2.c
index 59657168fd37..f2942e82f74f 100644
--- a/drivers/media/platform/imagination/vxe-vxd/encoder/vxe_v4l2.c
+++ b/drivers/media/platform/imagination/vxe-vxd/encoder/vxe_v4l2.c
@@ -838,10 +838,9 @@ static int vxe_open(struct file *file)
 
 	for (i = 0; i < ctx->cap_queue.fmt->num_planes; i++) {
 		ctx->cap_queue.size_image[i] = topaz_get_coded_buffer_max_size(NULL,
-									 ctx->cap_queue.fmt->fmt,
-									 ctx->cap_queue.width,
-									 ctx->cap_queue.height,
-									 &ctx->rc);
+						(enum img_standard) ctx->cap_queue.fmt->fmt,
+						ctx->cap_queue.width, ctx->cap_queue.height,
+						&ctx->rc);
 
 		ctx->cap_queue.bytesperline[i] = 0;
 	}
-- 
2.45.2

